main/webapp/modules/core/styles/index/open-project-ui.less

```diff
@@ -111,113 +111,168 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+table.list-table{
+    margin:0;
 }
 
-#metadata-body tr:nth-child(even) {background: #f2f2f2}
-#metadata-body tr:nth-child(odd) {background: #FFF}
+#metadata-body td {
+  padding: 1px;
+}
 
-#metadata-body table {
-  table-layout: fixed;
+#metadata-body tr:nth-child(even) {
+  background: #f2f2f2;
+}
+#metadata-body tr:nth-child(odd) {
+  background: #fff;
 }
 
 #metadata-body table > tbody > tr > td {
   word-wrap: break-word;
 }
 
 #metadata-body table > tbody > tr:nth-child(1) > th:nth-child(2) {
- width: 80%
- }
+  width: 80%;
+}
 
 #metadata-body table > tbody > tr:nth-child(1) > th:nth-child(3) {
- width: 4%
- }
-
+  width: 4%;
+}
 
 /* Making metadata table scrollable*/
-.scrollable-table{
-	max-height: 70vh;
-	overflow-y: auto;
-	overflow-x: hidden;
+.scrollable-table {
+  max-height: 70vh;
+  overflow-y: auto;
+  overflow-x: hidden;
 }
 
 /* Making metadata table header sticky and darker*/
-#metadata-body table > tbody > tr >th{
-	position: -webkit-sticky;
-	position:sticky;
-	top:0;
-	color:black;
-	background:#b7b7b7;
+#metadata-body table > tbody > tr > th {
+  position: -webkit-sticky;
+  position: sticky;
+  top: 0;
+  color: black;
+  background: #b7b7b7;
 }
 
- 
- /* Project Tags */
+/* Project Tags */
 #projectTags {
-	position: relative;
-	padding: 3px 10px;
-	margin: 5px;
-	font-size: 130%;
-	border-radius: 5px;
-	-moz-border-radius: 5px;
-	-webkit-border-radius: 5px;
-	background: #bcf;
-	min-height: 18px;
-	height: auto;
-	overflow:auto;
+  position: relative;
+  padding: 3px 10px;
+  margin: 0px;
+  font-size: 130%;
+  border-radius: 0px;
+  -moz-border-radius: 0px;
+  -webkit-border-radius: 0px;
+  background: #bcf;
+  min-height: 18px;
+  height: auto;
+  overflow: auto;
 }
 
 ul#tagsUl {
-	margin: 0;
-	padding: 0;
+  padding: 0;
+  left: 60px; // add left to make room for magnifying glass icon
+  display: inline-block;
 }
 
 ul#tagsUl li {
-	list-style: none;
-	float: left;
-	text-align: center;
-	min-width: 80px;
+  list-style: none;
+  float: left;
+  text-align: center;
+  min-width: 80px;
 }
 
 ul#tagsUl li a {
-	padding: 6px 5px; 
-	display : inline-block;
-	text-align: center;
-	color: #444;
-	text-shadow: 0 -1px 2px rgba(255, 255, 255, 0.60);
-	cursor: pointer;
-	font-weight: bold;
-	min-width: 50px;
+  padding: 6px 5px;
+  display: inline-block;
+  text-align: center;
+  color: #444;
+  text-shadow: 0 -1px 2px rgba(255, 255, 255, 0.6);
+  cursor: pointer;
+  font-weight: bold;
+  min-width: 50px;
 }
 
-ul#tagsUl li a  {
-	text-decoration: none;
+ul#tagsUl li a {
+  text-decoration: none;
 }
 
 #projectTags .current {
-	background-color: #DEE6FF;
-	border-radius: 10px;
-	-moz-border-radius: 10px;
-	-webkit-border-radius: 10px;
+  background-color: #dee6ff;
+  border-radius: 10px;
+  -moz-border-radius: 10px;
+  -webkit-border-radius: 10px;
+}
+/* Search bar in open project */
+// search
+#divSearch {
+  float: left;
+  width: 2.5%;
+  margin-left: -0.5%;
+}
+#divImage {
+  float: left;
+  width: 90%;
+}
```

Comment:
Those relative widths seem unreliable. On a large screen, you get a result like this:
![image](https://user-images.githubusercontent.com/309908/162733041-902618b6-d49c-41b4-bc96-eb8d0b0e3095.png)